buttons.css

```diff
@@ -0,0 +1,197 @@
+.flex-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+}
+.btn-contained {
+    background-color: #FFAC63;
+    color: black;
+    
+    padding: 10px 1rem;
+    text-transform: uppercase;
+    min-width: 64px;
+    height: 36px;
+    border: 0;
+    border-radius: 5px;
```

Comment:
just a suggestion try to use "rem" mostly instead of "px" in CSS. it will help you a lot in responsiveness
rest all is good !